samples/standalone/src/main/kotlin/org/jetbrains/jewel/samples/standalone/Main.kt

```diff
@@ -27,6 +28,10 @@ import org.jetbrains.jewel.window.styling.TitleBarStyle
 import java.io.InputStream
 
 fun main() {
+    JewelLogger
+        .getInstance("StandaloneSample")
+        .info("Starting Jewel Standalone showcase app")
```

Comment:
One bit missing from before:

```suggestion
        .info("Starting Jewel Standalone sample")
```

Bigger picture though, same as for the one in the IDE sample, I don't really see the value of printing this here. I understand it was done for debugging while developing the logger, but... do we still need it?